src/base/rss/rss_session.cpp

```diff
@@ -102,6 +102,24 @@ Session::Session()
         m_refreshTimer.start(m_refreshInterval * MsecsPerMin);
         refresh();
     }
+
+    // remove legacy/corrupted settings
+    auto settingsStorage = SettingsStorage::instance();
+    settingsStorage->removeValue("Rss/streamList");
+    settingsStorage->removeValue("Rss/streamAlias");
+    settingsStorage->removeValue("Rss/open_folders");
+    settingsStorage->removeValue("Rss/qt5/splitter_h");
+    settingsStorage->removeValue("Rss/qt5/splitterMain");
+    settingsStorage->removeValue("Rss/hosts_cookies");
+    settingsStorage->removeValue("RSS/streamList");
+    settingsStorage->removeValue("RSS/streamAlias");
+    settingsStorage->removeValue("RSS/open_folders");
+    settingsStorage->removeValue("RSS/qt5/splitter_h");
+    settingsStorage->removeValue("RSS/qt5/splitterMain");
+    settingsStorage->removeValue("RSS/hosts_cookies");
+    settingsStorage->removeValue("Rss/Session/EnableProcessing");
+    settingsStorage->removeValue("Rss/Session/RefreshInterval");
+    settingsStorage->removeValue("Rss/Session/MaxArticlesPerFeed");
```

Comment:
There's another key that's under the `RSS` group: `RSS/AutoDownloader/EnableProcessing`

Reply:
You're right!